File: Python Word Examples/ShapesApiDemo/deleteShape.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.ShapesApi(configuration)

file_name = 'Shape.docx'
paragraph_path = 'Section/0/Body/0/Paragraph/0'
index = 1
extra = {'folder': "input", 'dest_file_path': 'output/DeleteShape.docx'}
result = api.delete_shape(file_name, paragraph_path, index, **extra)

File: JavaScript Word Examples/ShapesApiDemo/deleteShape.js
(function (){
    var appId = "your id";
    var appKey = "your key";  
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.ShapesApi(apiClient);

    var fileName = "DeleteShape.docx";
    var paragraphPath = "Section/0/Body/0/Paragraph/0";
    var index = 1;
    var destFilePath = "output/DeleteShape_out.docx";
    var opts = { folder: 'input' };
    instance.deleteShape(fileName, paragraphPath, index, destFilePath, opts,
        function (error, data, response) {
            if (error) throw error;
        });
})();